6/s2i/bin/assemble

```diff
@@ -22,6 +22,12 @@ shopt -s dotglob
 echo "---> Installing application source ..."
 mv /tmp/src/* ./
 
+if [[ "$1" == "-h" ]]; then
+    # If the 'nodejs' assemble script is executed with '-h' flag,
```

Comment:
Is it even possible to run it with that flag via s2i?

Reply:
I am not so familiar with s2i, let me ask colleague or you how to test it.

Reply:
There doesn't seem to be any argument of `s2i build` to pass additional args to assemble script.

@pkubatrh +1, this should probably be removed (or moved to usage script)